asciidoctorj-core/src/main/java/org/asciidoctor/jruby/cli/SeverityConverter.java

```diff
@@ -0,0 +1,12 @@
+package org.asciidoctor.jruby.cli;
+
+import com.beust.jcommander.IStringConverter;
+import org.asciidoctor.SafeMode;
+import org.asciidoctor.log.Severity;
+
+public class SeverityConverter implements IStringConverter<Severity> {
+
+    @Override
+    public Severity convert(String argument) { return Severity.valueOf(argument.toUpperCase()); }
```

Comment:
When I pass an invalid option like `INFOX` I currently get a stacktrace back that does not show the currently supported options:
```
Exception in thread "main" java.lang.IllegalArgumentException: No enum constant org.asciidoctor.log.Severity.INFOX
	at java.base/java.lang.Enum.valueOf(Enum.java:273)
	at org.asciidoctor.log.Severity.valueOf(Severity.java:3)
	at org.asciidoctor.jruby.cli.SeverityConverter.convert(SeverityConverter.java:10)
...
```
What do you think about changing it so that you'll get just a message with the invalid value and all supported values like:
```
Could not convert severity 'INFOX'. Supported values are [DEBUG, INFO, WARN, ERROR, FATAL, UNKNOWN]
```

Nitpick: Could you please add newlines after the opening brace?

Reply:
I agree - and it turns out JCommander has `EnumConverter` class which gets us this nice error message for free.